### PR TITLE
Add elasticdump utility

### PR DESCRIPTION
**Pull request by MTN-RowinAndruscavage**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -5,22 +5,43 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
 
+- include: elasticdump.yml
```
Comment (senorsmile) [inline tasks/main.yml:8]: should we call this file elasticdump-debian.yml to conform to this repo's other includes, and leave room for someone to contribute a redhat version later?

File: tasks/main.yml [hunk 2]
```diff
@@ -5,22 +5,43 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
```
Comment (senorsmile) [inline tasks/main.yml:7]: should elasticdump be an optional install?

File: tasks/setup-Debian.yml [hunk 3]
```diff
@@ -12,3 +12,10 @@
     repo: 'deb https://artifacts.elastic.co/packages/5.x/apt stable main'
     state: present
     update_cache: yes
+
+- name: Install nodejs and friends for elasticdump
```
Comment (senorsmile) [inline tasks/setup-Debian.yml:16]: shouldn't this be in the elasticdump.yml include yml?

Comment (MTN-RowinAndruscavage) [reply tasks/setup-Debian.yml:16]: Added the nodejs + npm install to setup-RedHat, so all of this should make sense now.  Haven't had a chance to test it in Redhat/CentOS, though...

**Pull request merged**